### ISSUE TITLE
Localization missing for Add a menu -link

### ISSUE THREAD

**Issue by ronilaukkarinen** [digitoimistodude/air-light#64]
It seems we don't have a default localization for "Add a menu" -link. Let's add this in some point.

![Screen-Shot-2021-02-19-12-23-02](https://user-images.githubusercontent.com/1534150/108492231-93d43a00-72ad-11eb-8a2f-0ad43c31a51d.png)
